src/resources/views/character/mail-timeline.blade.php

```diff
@@ -77,7 +77,26 @@
             @endforeach
 
           </li>
-          @endif
+
+        @endif
+
+        @if($message->recipients->where('recipient_type', 'mailing_list')->count() > 0)
+
+          <li>
+            <b>{{ trans('web::seat.to_mailing_list') }}:</b>
+
+            @foreach($message->recipients->where('recipient_type', 'mailing_list') as $recipient)
+
+              {{ isset($MailMailingList::where('mailing_list_id', $recipient->recipient_id)->first()->name) ? $MailMailingList::where('mailing_list_id', $recipient->recipient_id)->first()->name : trans('web::seat.unknown') }}
+
+            @endforeach
+
+          </li>
+
+        @endif
+
+        <b>{{ trans('web::seat.source') }}: </b>
```

Comment:
you forget to push the language file.

Reply:
i didn't need to as that is already in the language file: https://github.com/eveseat/web/blob/master/src/lang/en/seat.php#L359